### ISSUE TITLE
Image-> Resize and Adjust for multiple pages

### ISSUE THREAD

**Issue by JRPX33** [stevencohn/OneMore#859]
I'm finding I need to run the Image-> Resize and Adjust command on mutiple pages to reduce the size of my notebooks.  But it looks like OneMore will only run this command against the current page, even if I have multiple pages selected.

Would be super cool if I could run this against all selected pages.

Thanks!
Jeff

**Comment by stevencohn** [comment]
Here's an alternative. The Resize dialog will remember the most recently used absolute width. So you can run the Resize command on the first page. For all subsequent pages, you can press Ctrl+Alt+R (replay last command) and just hit Enter. You still have to walk to each page.

Not sure there would be a big demand for this feature in the long run.

**Comment by JRPX33** [comment]
Fair enough.  I'll probably only need it once too, so Ctrl+Alt+R works for me!  
But it looks like it's Shift+Alt+R on my machine, not sure if that's something I did.

But this will work great unless others think this is important.

**Comment by stevencohn** [comment]
Ha! You're right. Of cousre, it's Shift+Alt+R.

Thanks for using OneMore and providing valuable feedback.
